pkgs/development/python-modules/ci-py/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchPypi, isPy27
+, pytest, pytestrunner, pytestCheckHook }:
+
+buildPythonPackage rec {
+  version = "1.0.0";
+  pname = "ci-py";
+
+  disabled = isPy27;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "12ax07n81vxbyayhwzi1q6x7gfmwmvrvwm1n4ii6qa6fqlp9pzj7";
+  };
+
+  checkInputs = [ pytest pytestrunner pytestCheckHook ];
```

Comment:
pytestrunner needs to be a nativeBuildInputs because it is a setup requires https://github.com/grantmcconnaughey/ci.py/blob/master/setup.py#L10. Very annoying that this has proliferated throughout the python ecosystem

Reply:
I should elaborate why. If you were to disable checks it would fail at setup always

Reply:
Thanks, I didn't realize that.  Fixed.